doc/contributing/PROCESS.md

```diff
@@ -174,6 +174,92 @@ To make a new OpenAssetIO release, follow this procedure.
   uploaded correctly.
 - You're done! Take a break and relax!
 
+## Breaking Integrations
```

Comment:
[consider][minor] We have a bit of a mixture of capitalisation style for titles in this doc. Mostly we use "Capitalise first word only".  Would be good to be consistent.

Reply:
Fixed the rest of the document as well, there were a couple. [a0b6406](https://github.com/OpenAssetIO/OpenAssetIO/pull/772/commits/a0b6406421d8641b3ef902737ba9847e55ad5ff9)